### PR TITLE
Added neonblueflame-MOTHERSHIP.md to class repo

### LINKED ISSUE #15: neonblueflame tracks progress on GitHub Flow
## Tasks
- [x] create a **branch** (git)
- [x] add some **commits** (git)
- [x] open a _pull request_ (GitHub)
- [ ] additional **commits** (git)
- [ ] _merge_ (git)

### PR DESCRIPTION
**Pull request by neonblueflame**
This PR adds neonblueflame-MOTHERSHIP.md to class repo. Closes #15.

- Do you like :pizza: ?
- What type of :pizza: do you like?

### CODE REVIEW

File: neonblueflame-MOTHERSHIP.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+
+
+### Attractions for tourists
+
+- VirtualBox Linux VMs, I guess
+
+
+### Attractions for locals
```
Comment (shamusharkin) [inline neonblueflame-MOTHERSHIP.md]: what do the locals do for fun? Unix?

Comment (neonblueflame) [inline neonblueflame-MOTHERSHIP.md]: :+1: 
Actually, windows ;)

Comment (neonblueflame) [inline neonblueflame-MOTHERSHIP.md]: @shamusharkin I've added your suggestion :)

**Comment by neonblueflame** [comment]
I'll ship with 1 :+1: , :ship: , :shipit:

**Comment by neonblueflame** [comment]
Yey!

**Pull request merged**